doc/classes/Input.xml

```diff
@@ -125,6 +169,18 @@
 				On Windows, all XInput joypad GUIDs will be overridden by Godot to [code]__XINPUT_DEVICE__[/code], because their mappings are the same.
 			</description>
 		</method>
+		<method name="get_joy_gyroscope" qualifiers="const">
+			<return type="Vector3" />
+			<param index="0" name="device" type="int" />
+			<description>
+			</description>
+		</method>
+		<method name="get_joy_gyroscope_enabled" qualifiers="const">
```

Comment:
Methods that return a boolean should use an `is_` prefix instead of `get_`:

```suggestion
		<method name="is_joy_gyroscope_enabled" qualifiers="const">
```